fixtures/webview.py

```diff
@@ -85,3 +85,22 @@ def webview_instance(webview_base_url):
         return url.subdomain
     else:
         return "prod"
+
+
+@pytest.fixture
+def redirecting_books_titles():
+    """Returns all the books from the list of redirecting books to rex
+    """
+
+    here = os.path.join(DATA_DIR, "redirected_books.txt")
+
+    f = open(here, "r")
```

Comment:
It's a best practice in python to use a context manager. there are other examples of this in the framework. You'll want to implement this like:

`with open(filename, "mode") as file:` ...